### DISCUSSION TITLE
CobraPin Servo example code

### DISCUSSION THREAD

**Discussion by DorusvdLinden** [missionpinball/missionpinball#92] [Q&A] [answered]
Dear all,

Does anyone have a MPF code example for connecting a servo to the CobraPin controller? 

I am building a ball dropper with a valve I would like to open with a servo.
3d design: 
Thank you
![2023-04-17 (2)](https://user-images.githubusercontent.com/46962795/232566704-82ccf510-6ee7-4f81-90ef-8374ececae29.png)

link to design in OnShape: 
https://cad.onshape.com/documents/8212fe3e618a0fe47cd2c238/w/885dc9fd1c559656144e3042/e/d3f79d14fcaf02d7c5b00654?renderMode=0&uiState=643d84e35b46bd01230e6c34

![2023-04-17 (3)](https://user-images.githubusercontent.com/46962795/232567109-995e7d05-1cff-49a8-bd83-0ae996709bff.png)

**Comment by agrisdul** [comment]
Hi, did you checked here: https://github.com/missionpinball/mpf-docs/blob/main/hardware/pololu_maestro/index.rst  same is for the stepper motors in mpf-docs folder

**Comment by cobra18t** [comment, marked as answer]
I saw this discussion first over on the MPF Users Google Group, so see the responses there if interested: https://groups.google.com/g/mpf-users/c/DbKRG73KCx4/m/PcBQx5taCgAJ?utm_medium=email&utm_source=footer
